runner/src/index.ts

```diff
@@ -1,9 +1,10 @@
 import createServer from "./server";
+import logger from "./server/plugins/logger";
 
 createServer({})
   .then((server) => server.start())
   .then(() => process.send && process.send("online"))
   .catch((err) => {
-    console.error(err);
+    logger.error(err);
```

Comment:
you can just use console.error here, as suggested by hapi-pino, https://github.com/pinojs/hapi-pino#usage